node.rb

```diff
@@ -0,0 +1,4 @@
+module Raft
+  class Node
```

Comment:
What do you think about adding a docstring here briefly explaining what a node represents? It can even be a verbatim copy of some definition from the Raft paper or prior art. It would be useful to give the context as the class start to do things

Reply:
great suggestion, added 👍